SampleUsages/TestScenario.cs

```diff
@@ -27,7 +27,7 @@ class TestScenario
         public int WarmUpTimeInMinutes { get; set; }
         private ILogger _logger { get; set; }
 
-        internal void RunScenario(ILogger logger)
+        internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario testScenario = null)
         {
             this._logger = logger;
             var scenarioType = Tuple.Create(this.Platform, this.TriggerType);
```

Comment:
If we call this method "RunTest" then to keep the naming consistent maybe we can do the following scenarioType renamed to testType